.github/workflows/_check-code-formatting.yml

```diff
@@ -0,0 +1,27 @@
+---
+name: Checks code formatting
+
+on:
+  workflow_call:
+
+jobs:
+  main:
+    name: Run check, test and lints
+    runs-on: [self-hosted, Linux, X64, large]
+    env:
+      CARGO_INCREMENTAL: 0
+      RUSTC_WRAPPER: sccache
+      RUST_TOOLCHAIN: nightly-2023-01-10
```

Comment:
I've created repository variable `RUST_TOOLCHAIN_NIGHTLY` with value `nightly-2023-01-10`. We can use it in CI context, e.g. in this file as `${{ vars.RUST_TOOLCHAIN_NIGHTLY }}`